wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/Receivers/Building_Grid.groovy

```diff
@@ -139,6 +139,9 @@ def exec(Connection connection, input) {
 
 
     String receivers_table_name = "RECEIVERS"
+    if (input["receiversTableName"]) {
```

Comment:
There is no input named receiversTableName

Reply:
dommage, mais ok